README.md

```diff
@@ -1,13 +1,16 @@
+[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-24ddc0f5d75046c5622901739e7c5dd533143b0c8e959d652212380cedb1ea36.svg)](https://classroom.github.com/a/fbkbKZ5N)
 # Asymptotic Equivalences
 
 In the lectures, we said that logarithms with different bases don't affect the
 asymptotic complexity of an algorithm. Prove that $O(\log_{2} n)$ is the same as
 $O(\log_{5} n)$. Use the mathematical definition of $O$ -- do a formal proof,
 not just the intuition.
 
-I have started with the formal definition of $O$ below. Add your answer to this
-markdown file. [This
-page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
-might help with the notation for mathematical expressions.
-
-$T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
+$T(n) \in O(g(n)) \iff (\exists c)(\exists n_0) T(n) \leq c g(n) (\forall n \geq n_0)$  
+$f(n) \in O(\log_2 n) \iff (\exists c)(\exists n_0)(\forall n \geq n_0) f(n) \leq c \log_2 n$  
+$\log_m n = \frac{\ln n}{\ln m}$  
+$c := \ln 2 \qquad f(n) \in O(\log_2 n) \iff (\exists n_0)(\forall n \geq n_0) f(n) \leq \ln n$  
```

Comment:
How do you know that $c$ can be $\ln 2$?

Reply:
I have updated my analysis to explain how I choose my values for $c$ and why I believe such a value must exist to satisfy the $O(g(n))$ definition.